### ISSUE TITLE
Upload CLI jar to repository

### ISSUE THREAD

**Issue by vromero** [aml-org/amf#334]
The CLI jar, meant to be a quick and handy tool has its whole point defeated by the fact that the only way to obtain it is to download the sources and compiling.

**Comment by kevinduffey** [comment]
I agree. I do not (nor can not in some cases) install the full SBT stack just to build the jar file. This should be built and pushed as a maven artifact, as well as a download. I would even say, today, besides a maven artifact for Java developers to use in their own code, pushing a Docker image that can be used in CLI mode may be useful, as this is something we would use during a build step most likely.
